.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+##  :notebook_with_decorative_cover: Description
+
+:pencil2:
+
+## :dart: Review Instructions
+
+Please review this pull request as follows:
+
+- [ ] Commit-by-commit
+- [ ] File-by-file
```

Comment:
```suggestion
##  :notebook_with_decorative_cover: Description

<!-- Describe the change you've made in this section -->

##  :memo: Checklist

- [ ] All user-facing changes have changelog entries.
- [ ] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast).
- [ ] The PR description contains instructions for the reviewer, if necessary.

## :dart: Review Instructions

<!-- Provide instructions for the reviewer here, e.g., review this pull request commit-by-commit, or file-by-file. -->
```

Reply:
I like the checkbox thing better (last section). It is easier when the "instructions" are always formatted the same way, i.e., a checkbox. If you want to put clear text you can still add that, right?

Reply:
The reason why I removed the checkboxes from that section is that the PR list view shows remaining tasks, and having a checklist there would mean that we always have one outstanding task for every PR.

Reply:
Good point, unfortunately that prevents the checkboxes. We could just a add bullet list that users adapt to their needs, e.g., 

```markdown
<!-- Provide review instructions here. -->

- Commit-by-commit
- File-by-file
```

Reply:
I'd be fine with either.